x/evm/types/events.go

```diff
@@ -9,6 +9,7 @@ const (
 	EventTypeTokenConfirmation             = "tokenConfirmation"
 	EventTypeTransferKeyConfirmation       = "transferKeyConfirmation"
 	EventTypeLink                          = "link"
+	EventTypeStartedSigning                = "startedSigning"
```

Comment:
```suggestion
	EventTypeStartSigning                = "startSigning"
```
nit: the past tense seems superfluous and not really within the current conventions